hw1/02-palindrome.html

```diff
@@ -6,20 +6,31 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
-    <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
+  <body class="bg-primary">
+    <main
+      class="container min-vh-100 d-flex align-items-center justify-content-center"
```

Comment:
I think this `align-items-center` will bring the item vertically centered, which is not what the picture looks like. Maybe changes this list a bit ❓

Reply:
correctly mentioned making it different from the screenshot